src/PersistentOrderedMap.mo

```diff
@@ -527,6 +527,50 @@ module {
       combine : (K, Value, Accum) -> Accum
     ) : Accum
     = Internal.foldRight(map, base, combine);
+
+    /// Test whether all key-value pairs satisfy a given predicate `pred`.
+    ///
+    /// Example:
+    /// ```
+    /// import Map "mo:base/PersistentOrderedMap";
+    /// import Nat "mo:base/Nat";
+    /// import Debug "mo:base/Debug";
+    ///
+    /// let mapOps = Map.MapOps<Nat>(Nat.compare);
+    /// let map = mapOps.fromIter<Text>(Iter.fromArray([(0, "0"), (2, "2"), (1, "1")]));
+    ///
+    /// Debug.print(debug_show(mapOps.all(map, func (k, v) = (v == debug_show(k)))));
```

Comment:
Can you give negative example here and positive for `some`? BTW, I didn't know that `=`-syntax works for lambdas.

Reply:
It works, when you have one expression. So you have two variants:
* `func x = x`
* func x { x }`

Reply:
fixed

Reply:
Fix the syntax you can check tests for `all` for example